grails-app/services/org/pih/warehouse/report/ReportService.groovy

```diff
@@ -1133,4 +1134,33 @@ class ReportService implements ApplicationContextAware {
 
         return new PaginatedList<InventoryAuditSummary>(data, results.totalCount);
     }
-}
\ No newline at end of file
+
+    def getTotalCount(IndicatorCommand command) {
```

Comment:
I know it's only mocked result, but anyway you know the result type - it's `Map`, it doesn't have to be `def`.